src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +60,46 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length == 0) {
+            return Optional.empty();
+        }
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (DateTimeException e) {
+            return Optional.empty();
```

Comment:
let's put it after catch block, we should not use try catch as if else(to build logic on it)